plugins/redis/redis.cpp

```diff
@@ -354,7 +354,7 @@ void * Connection::redisCommand(const char * format, ...)
 }
 int Connection::setTimeout(unsigned _timeout)
 {
-    struct timeval to = { _timeout/1000, (_timeout%1000)*1000 };
+    struct timeval to = { (int) _timeout/1000, (int) (_timeout%1000)*1000 };
```

Comment:
Getting a sense of deja vu here! I think we had a similar discussion last time I fixed this...